target/linux/rockchip/patches-6.6/401-1-arm64-dts-rockchip-rk3566-Add-Nanopi-R3S.patch

```diff
@@ -0,0 +1,567 @@
+--- /dev/null
```

Comment:
please renumber this to 127, i.e. `target/linux/rockchip/patches-6.6/127-arm64-dts-rockchip-rk3566-Add-Nanopi-R3S.patch`